docs/getting_started.md

```diff
@@ -0,0 +1,280 @@
+# Getting Started
```

Comment:
```suggestion
# Getting started
```

Reply:
as before, I would like to keep page titles capitalized